src/components/Attachment/__tests__/Card.test.js

```diff
@@ -80,6 +80,7 @@ describe('Card', () => {
             <a
               className="str-chat__message-attachment-card--url"
               href="https://google.com"
+              rel="noopener noreferrer"
```

Comment:
i'm not familiar with this, what does this mean?

Reply:
security:
rel="noopener" -Requires that any browsing context created by following the hyperlink must not have an opener browsing context
rel="noreferrer" -Requires that the browser should not send an HTTP referer header if the user follows the hyperlink

Pops up as a console warning if not included if the target attribute is set.